javalib/src/main/scala/java/util/ArrayDeque.scala

```diff
@@ -18,14 +18,13 @@ import java.lang.Utils._
 import java.util.ScalaOps._
 
 import scala.scalajs.js
+import scala.scalajs.LinkingInfo
```

Comment:
This import is not used anymore.

And I guess the import `scala.scalajs.js` isn't used anymore either.